hazelcast-client/src/test/java/com/hazelcast/client/impl/ClientTestUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.client.impl;
+
+import com.hazelcast.core.HazelcastInstance;
+import org.junit.Ignore;
+
+@Ignore("not a JUnit test")
```

Comment:
I copied it from our TestUtil class. If it is not necessary I can remove it. But I thought that there is a reason to add to TestUtil and I did the same here.